test/unit/middleware/spec.headers.js

```diff
@@ -19,18 +19,22 @@ describe('headers middleware', () => {
             },
             compatibility: {
                 middleware: sinon.stub().returns('compatibility middleware')
-            }
+            },
+            helmet: sinon.stub()
         };
 
+        stubs.helmet.frameguard = stubs.frameguard;
```

Comment:
This is a little weird, but I didn't want to complicate the tests by creating a full test double of the helmet library.

Reply:
I suppose `stubs.helmet.frameguard = sinon.stub().returns('frameguard middleware');` and removing line 15 would probably make more sense.